scripts/soltest.sh

```diff
@@ -1,12 +1,32 @@
 #!/usr/bin/env bash
-
 set -e
 
 REPO_ROOT="$(dirname "$0")"/..
 USE_DEBUGGER=0
 DEBUGGER="gdb --args"
 BOOST_OPTIONS=
 SOLTEST_OPTIONS=
+BUILD_DIR=${BUILD_DIR:-build}
```

Comment:
Isn't `BUILD_DIR` taken from the "global" environment a little too generic?

Reply:
Make a suggestion. For examample would  `SOLIDITY_BUILD_DIR` be better instead?

Reply:
I think ` SOLIDITY_BUILD_DIR` is better since it could be easily confused with the soltest build directory in this context.

Reply:
Ok - .8f1fcba40 uses `SOLIDITY_BUILD_DIR` now.